config/vendor-api-v1.yml

```diff
@@ -547,7 +549,7 @@ components:
           example: full_time
     Offer:
       type: object
-      additionalProperties: false
+      additionalProperties: true
```

Comment:
We've preferred to use `nullable: true` on the optional fields elsewhere for 2 reasons I can think of:

- it's better UX as clients know exactly what to expect
- our specs are more robust, as if something unexpected ends up here then the spec will fail

Would it be better in any case to always include the offer if present? Less for clients to think about

Reply:
ok! 👍

Reply:
Check the last commit, have done this

Reply:
Check the last commit, have done this